test/e2e/specs/homepage.js

```diff
@@ -398,5 +398,18 @@ module.exports = {
       .useXpath()
       .waitForElementVisible("//div[contains(@class, 'tooltip-inner') and text() = 'Nothing matched your search']")
     browser.end()
+  },
+
+  'it should contains a dropdown allowing to filter transactions types': function (browser) {
+    browser
+      .assert.urlContains('/#')
```

Comment:
don't do this, the `#` won't be there. Just remove the assert line altogether

Reply:
ok my bad, it does use the `#` but your test will fail since the previous test calls `browser.end()` to close the browser. So either remove it from there and add it to this test, or start the browser again on a specified URL